tests/virtualenv_harness.sh

```diff
@@ -56,5 +56,4 @@ fi
 
 make requirements
 
-run_tests $PROJECT_NAME $MODULE_NAME
-
+run_tests $PROJECT_NAME $MODULE_NAME
```

Comment:
Per other comment about having files end in a newline, this file shouldn't have any changes.

Reply:
@jayqi the file on master has two newlines at the end of it, so technically this change is making the file correctly formatted. Can move this change to a separate formatting PR though if desired.